crates/parser2/src/parser/expr.rs

```diff
@@ -325,6 +351,27 @@ fn is_lt_eq<S: TokenStream>(parser: &mut Parser<S>) -> bool {
 fn is_gt_eq<S: TokenStream>(parser: &mut Parser<S>) -> bool {
     parser.dry_run(|parser| parser.parse(GtEqScope::default(), None).0)
 }
+fn is_aug<S: TokenStream>(parser: &mut Parser<S>) -> bool {
+    parser.dry_run(|parser| {
+        if !bump_aug_assign_op(parser) {
+            return false;
+        }
+
+        parser.set_newline_as_trivia(false);
+        parser.current_kind() == Some(SyntaxKind::Eq)
+    })
+}
+fn is_asn<S: TokenStream>(parser: &mut Parser<S>) -> bool {
+    parser.dry_run(|parser| {
+        parser.set_newline_as_trivia(false);
+        if parser.current_kind() == Some(SyntaxKind::Eq) {
+            parser.bump();
+            true
+        } else {
+            false
+        }
+    })
+}
```

Comment:
```suggestion

fn is_aug<S: TokenStream>(parser: &mut Parser<S>) -> bool {
    parser.dry_run(|parser| {
        if !bump_aug_assign_op(parser) {
            return false;
        }

        parser.set_newline_as_trivia(false);
        parser.current_kind() == Some(SyntaxKind::Eq)
    })
}

fn is_asn<S: TokenStream>(parser: &mut Parser<S>) -> bool {
    parser.dry_run(|parser| {
        parser.set_newline_as_trivia(false);
        if parser.current_kind() == Some(SyntaxKind::Eq) {
            parser.bump();
            true
        } else {
            false
        }
    })
}
```
nit: please add newlines.